.changeset/friendly-terms-love.md

```diff
@@ -0,0 +1,253 @@
+'@backstage/plugin-github-issues': patch
+'@backstage/plugin-kafka-backend': patch
+'@backstage/plugin-nomad-backend': patch
+'@backstage/plugin-proxy-backend': patch
+'@backstage/plugin-search-common': patch
+'@backstage/plugin-tech-insights': patch
+'@backstage/plugin-techdocs-node': patch
+'@backstage/plugin-user-settings': patch
+'@backstage/plugin-vault-backend': patch
+'@backstage/integration': patch
+'@backstage/plugin-auth-backend': patch
+'@backstage/plugin-azure-devops': patch
+'@backstage/plugin-catalog-node': patch
+'@backstage/plugin-code-climate': patch
+'@backstage/plugin-gcp-projects': patch
+'@backstage/plugin-search-react': patch
+'@backstage/plugin-todo-backend': patch
+'@backstage/cli-common': patch
+'@backstage/create-app': patch
+'@backstage/repo-tools': patch
+'@backstage/test-utils': patch
+'@backstage/plugin-adr-backend': patch
+'@backstage/plugin-app-backend': patch
+'@backstage/plugin-azure-sites': patch
+'@backstage/plugin-events-node': patch
+'@backstage/plugin-firehydrant': patch
+'@backstage/dev-utils': patch
+'@backstage/plugin-adr-common': patch
+'@backstage/plugin-cloudbuild': patch
+'@backstage/plugin-home-react': patch
+'@backstage/plugin-kubernetes': patch
+'@backstage/plugin-lighthouse': patch
+'@backstage/plugin-scaffolder': patch
+'@backstage/plugin-stackstorm': patch
+'@backstage/plugin-tech-radar': patch
+'@backstage/plugin-vault-node': patch
+'@backstage/cli-node': patch
+'@backstage/codemods': patch
+'@backstage/plugin-auth-node': patch
+'@backstage/plugin-codescene': patch
+'@backstage/plugin-dynatrace': patch
+'@backstage/plugin-gcalendar': patch
+'@backstage/plugin-org-react': patch
+'@backstage/plugin-pagerduty': patch
+'@backstage/plugin-shortcuts': patch
+'@backstage/plugin-sonarqube': patch
+'@backstage/plugin-xcmetrics': patch
+'@backstage/plugin-airbrake': patch
+'@backstage/plugin-api-docs': patch
+'@backstage/plugin-app-node': patch
+'@backstage/plugin-circleci': patch
+'@backstage/plugin-devtools': patch
+'@backstage/plugin-graphiql': patch
+'@backstage/plugin-linguist': patch
+'@backstage/plugin-newrelic': patch
+'@backstage/plugin-opencost': patch
+'@backstage/plugin-periskop': patch
+'@backstage/plugin-playlist': patch
+'@backstage/plugin-puppetdb': patch
+'@backstage/plugin-techdocs': patch
+'@backstage/config': patch
+'@backstage/errors': patch
+'@backstage/plugin-bitrise': patch
+'@backstage/plugin-catalog': patch
+'@backstage/plugin-explore': patch
+'@backstage/plugin-jenkins': patch
+'@backstage/plugin-rollbar': patch
+'@backstage/theme': patch
+'@backstage/types': patch
+'@backstage/plugin-allure': patch
+'@backstage/plugin-badges': patch
+'@backstage/plugin-bazaar': patch
+'@backstage/plugin-search': patch
+'@backstage/plugin-sentry': patch
+'@backstage/plugin-fossa': patch
+'@backstage/plugin-ilert': patch
+'@backstage/plugin-kafka': patch
+'@backstage/plugin-nomad': patch
+'@backstage/plugin-vault': patch
+'@backstage/cli': patch
+'@backstage/plugin-gocd': patch
```

Comment:
reminder to remove everything from here except repo-tools since that's the one with actual shippable changes